src/Core/src/Core/ISearchBar.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Microsoft.Maui
+{
+    public interface ISearchBar : IView
```

Comment:
Should this maybe inherit from `IText` as well and not have the Text` property? For the additional properties, just let then exist and only add the mapping for text in this PR. The rest can be done later and we probably can and should re-use the code from Entry.